mkdocs.yml

```diff
@@ -13,6 +13,7 @@ theme:
   features:
     - content.action.edit
     - content.action.view
+    - content.tabs.link
```

Comment:
Is this the change needed to have the tabs change in sync? Nice one! Thanks!

Reply:
Correct